source/framework/core/inc/TRestEvent.h

```diff
@@ -105,6 +105,6 @@ class TRestEvent : public TObject {
     // Destructor
     virtual ~TRestEvent();
 
-    ClassDef(TRestEvent, 1);  // REST event superclass
+    ClassDef(TRestEvent, 2);  // REST event superclass
```

Comment:
No need to change the ClassDef version if metadata members have not been changed.

Reply:
OK, I am never sure of this.